analysis/topEFT/get_yield_json.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import json
+import datetime
+from topcoffea.modules.YieldTools import YieldTools
+import topcoffea.modules.MakeLatexTable as mlt
+
+# This script takes a pkl file, finds the yields in the analysis categories, saves the yields to a json
+# Example usage:
+# python get_yield_json.py histos/plotsTopEFT.pkl.gz
```

Comment:
I've expanded the description a bit, and as we discussed in our conversation earlier, it would be difficult to format the text in an easily readable way as a description in `ArgumentParser()`, so we'll just plan to leave it as is for now.